packages/runtime/src/modules/RequestModule.ts

```diff
@@ -102,7 +102,7 @@ export class RequestModule extends RuntimeModule {
             }
 
             this.logger.info(
-                `There is already an open Relationship for the RelationshipTemplate '${template.id}' and onExistingRelationship is not defined. Skipping creation of a new request.`
+                `There is already an active Relationship to the creator of the RelationshipTemplate '${template.id}' and an onExistingRelationship request is not defined. Skipping creation of a new request.`
```

Comment:
If you adjust this, you could also capitalize "Request".

Reply:
You're right, if I revise the log entry anyway... :D